serverless/functions/serverless-functions-on-cluster-builds.adoc

```diff
@@ -0,0 +1,92 @@
+:_content-type: PROCEDURE
+[id="serverless-functions-on-cluster-builds"]
+= Building and deploying functions on-cluster
+:context: serverless-functions-on-cluster-builds
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+You can use the Knative (`kn`) CLI to push a function project directly to the cluster, which triggers a build and deployment of the function. You can also push a function project to a source code repository branch, which triggers a build of the function with the updated changes and deploys the updated function on the cluster.
```

Comment:
This wording is a little imprecise. The CLI does not actually push a project to the cluster. It instead only creates a Pipeline on the cluster which pulls the function project source from GitHub. Pushing code directly to the cluster is a desired capability but not implemented yet. It is tracked here: https://github.com/knative-sandbox/kn-plugin-func/issues/952 

Perhaps rephrasing to something like this would be clearer.
```suggestion
You can use the Knative (`kn`) CLI to initiate a function project build directly on the cluster. Building a function project in this way requires that your function project source code exist in a Git repository branch accessible to the cluster.
```